templates/v2/vault/standard_layout.html.twig

```diff
@@ -2,6 +2,6 @@
 
 {% block body %}
     {% set beneficiary = beneficiary is defined ? beneficiary : null %}
-    {{ render(controller('App\\ControllerV2\\HomeController::renderBeneficiaryNav', {beneficiary})) }}
+    {{ render(controller('App\\ControllerV2\\HomeController::renderBeneficiaryNav', { 'beneficiary': beneficiary, 'currentRoute': app.request.attributes.get('_route') })) }}
```

Comment:
`{{ render(controller('App\\ControllerV2\\HomeController::renderBeneficiaryNav', {beneficiary: beneficiary, currentRoute: app.request.attributes.get('_route')})) }}` It doesn't work this way without single quotes ?